python/thrift/appengine-ssl-gateway/README.md

```diff
@@ -0,0 +1,253 @@
+# Connecting from Google App Engine to Google Cloud Bigtable
+
+It's easy to deploy a Thrift gateway cluster on Google Compute Engine that
+proxies request to the Cloud Bigtable API.
+
+Here are step by step instructions for starting a Cloud Bigtable cluster and
+building and deploying a Thrift server on Compute Engine.
+
+**Table of contents**
```

Comment:
While probably not a huge issue for the customers you are targeting this at, you should have a requirements section.  

`make` is on most systems, but not all, many Java folks don't use it.